core/store/src/store.rs

```diff
@@ -1,19 +1,63 @@
-use std::fs::File;
-use std::path::Path;
-use std::sync::Arc;
-use std::{fmt, io};
-
-use borsh::{BorshDeserialize, BorshSerialize};
-use near_fmt::{AbbrBytes, StorageKey};
-
 use crate::DBCol;
 use crate::adapter::{StoreAdapter, StoreUpdateAdapter};
 use crate::db::metadata::{DbKind, DbMetadata, DbVersion, KIND_KEY, VERSION_KEY};
 use crate::db::{DBIterator, DBOp, DBSlice, DBTransaction, Database, StoreStatistics, refcount};
+use borsh::{BorshDeserialize, BorshSerialize};
+use near_fmt::{AbbrBytes, StorageKey};
+use std::fs::File;
+use std::path::Path;
+use std::sync::Arc;
+use std::{fmt, io};
 
 const STATE_COLUMNS: [DBCol; 2] = [DBCol::State, DBCol::FlatState];
 const STATE_FILE_END_MARK: u8 = 255;
 
+pub(super) mod deserialized_column {
+    use crate::DBCol;
+    use std::any::Any;
+    use std::sync::Arc;
+
+    type CacheContainer = near_cache::SyncLruCache<Vec<u8>, Arc<dyn Any + Send + Sync>>;
```

Comment:
(no action needed) if we use an LRU here, it requires gets to synchronize (to update the position of the recently accessed item on `get`).
This is why I sometimes prefer using a FIFO cache which allows concurrent gets, although I don't know how much concurrent gets are used in the codebase.